api/users/views.py

```diff
@@ -395,6 +397,8 @@ class UserAddonAccountList(JSONAPIBaseView, generics.ListAPIView, UserMixin, Add
     view_category = 'users'
     view_name = 'user-external_accounts'
 
+    ordering = ('-date_last_refreshed',)
```

Comment:
This should be changed to `date_modified` after https://openscience.atlassian.net/browse/OSF-7968 is implemented, but this is fine for now.